### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by jasonwalker80** [dhspence/docker-haloplex_demux#2]
Biopython is missing:
```
Traceback (most recent call last):
  File "/usr/bin/demultiplexer_hja_bd.py", line 9, in <module>
    from Bio.SeqIO.QualityIO import FastqGeneralIterator
ImportError: No module named Bio.SeqIO.QualityIO
```

It looks like https://github.com/dhspence/docker-haloplex_demux/pull/1 will fix this but needs to be merged back to master.

Also, I'd suggest adding libnss-sss to the list of installs so a user is properly authenticated on MGI hosts.

**Comment by jasonwalker80** [comment]
This should be resolved by #3

**Comment by jasonwalker80** [comment]
Merged. resolving.
